### PR TITLE
[OCPBUGS-24484] Add missing launchSecurity parameter and verification

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.14 +

Issue: https://issues.redhat.com/browse/OCPBUGS-24484

Link to docs preview:
https://69341--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z-kvm#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by clapolt** [comment]
Section 4.
the --launchSecurity type="s390-pv" line in the virt-install example should have "\" at the end, yes?

**Comment by clapolt** [comment]
LGTM

**Comment by nikita-dubrovskii** [comment]
LGTM

File: modules/ibm-z-secure-execution.adoc [hunk 1]
```diff
@@ -112,13 +112,19 @@ To achieve the desired security, Ignition logging and local login are disabled b
 ----
 gpg --recipient-file /path/to/ignition.gpg.pub --yes --output /path/to/config.ign.gpg --verbose --armor --encrypt /path/to/config.ign
 ----
+
+. Follow the fast-track installation procedure to install nodes using the {ibm-name} Secure Execution QCOW image.
 +
 [NOTE]
 ====
-Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` when mounting the Ignition file.
+Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` and add the `launchSecurity` parameter when mounting the Ignition file.
```
Comment (nikita-dubrovskii) [inline modules/ibm-z-secure-execution.adoc:120]: `launchSecurity` affects all IO deives and must be enabled on each VM boot. I guess it'd be better to drop `when mounting the Ignition file.`

**Comment by holgwolf** [comment]
LGTM

File: installing/installing_ibm_z/installing-ibm-z-kvm.adoc [hunk 2]
```diff
@@ -97,6 +97,8 @@ include::modules/ibm-z-secure-execution.adoc[leveloffset=+2]
 
 * link:https://www.ibm.com/docs/en/linux-on-systems?topic=ibmz-secure-execution[Linux as an {ibm-name} Secure Execution host or guest]
 
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/securing-virtual-machines-in-rhel_configuring-and-managing-virtualization#setting-up-secure-execution-on-ibm-z_securing-virtual-machines-in-rhel[{op-system-base-full} 9 - Setting up {ibm-name} Secure Execution on {ibm-z-title}]
```
Comment (agantony) [inline installing/installing_ibm_z/installing-ibm-z-kvm.adoc:100]:
```suggestion
* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/securing-virtual-machines-in-rhel_configuring-and-managing-virtualization#setting-up-secure-execution-on-ibm-z_securing-virtual-machines-in-rhel[Setting up {ibm-name} Secure Execution on {ibm-z-title}]
```

File: installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc [hunk 3]
```diff
@@ -104,6 +104,8 @@ include::modules/ibm-z-secure-execution.adoc[leveloffset=+2]
 
 * link:https://www.ibm.com/docs/en/linux-on-systems?topic=ibmz-secure-execution[Linux as an {ibm-name} Secure Execution host or guest]
 
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/securing-virtual-machines-in-rhel_configuring-and-managing-virtualization#setting-up-secure-execution-on-ibm-z_securing-virtual-machines-in-rhel[{op-system-base-full} 9 - Setting up {ibm-name} Secure Execution on {ibm-z-title}]
```
Comment (agantony) [inline installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc:107]: same as above

File: modules/ibm-z-secure-execution.adoc [hunk 4]
```diff
@@ -112,13 +112,19 @@ To achieve the desired security, Ignition logging and local login are disabled b
 ----
 gpg --recipient-file /path/to/ignition.gpg.pub --yes --output /path/to/config.ign.gpg --verbose --armor --encrypt /path/to/config.ign
 ----
+
+. Follow the fast-track installation procedure to install nodes using the {ibm-name} Secure Execution QCOW image.
```
Comment (agantony) [inline modules/ibm-z-secure-execution.adoc:116]:
```suggestion
. Follow the fast-track installation of RHCOS to install nodes by using the {ibm-name} Secure Execution QCOW image.
```

File: modules/ibm-z-secure-execution.adoc [hunk 5]
```diff
@@ -112,13 +112,19 @@ To achieve the desired security, Ignition logging and local login are disabled b
...
 +
 [NOTE]
 ====
-Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` when mounting the Ignition file.
+Before starting the VM, replace `serial=ignition` with `serial=ignition_crypted` and add the `launchSecurity` parameter.
```
Comment (agantony) [inline modules/ibm-z-secure-execution.adoc:120]:
```suggestion
Before you start the VM, replace `serial=ignition` with `serial=ignition_crypted`, and add the `launchSecurity` parameter.
```

File: modules/ibm-z-secure-execution.adoc [hunk 6]
```diff
@@ -112,13 +112,19 @@ To achieve the desired security, Ignition logging and local login are disabled b
...
 ====
-+
-When Ignition runs on the first boot, and the decryption is successful, you will see an output like the following example:
+
+.Verification
+
+When you have completed the fast-track installation procedure and Ignition runs on the first boot, verify if decryption is successful.
```
Comment (agantony) [inline modules/ibm-z-secure-execution.adoc:125]:
```suggestion
* Verify if decryption was successful after the ignition was executed at the first boot.
```

File: modules/ibm-z-secure-execution.adoc [hunk 7]
```diff
@@ -112,13 +112,19 @@ To achieve the desired security, Ignition logging and local login are disabled b
...
+
+* If the decryption is successful, you will see an output like the following example:
```
Comment (agantony) [inline modules/ibm-z-secure-execution.adoc:127]:
```suggestion
** If the decryption was successful, you can expect an output similar to the following example:
```

File: modules/ibm-z-secure-execution.adoc [hunk 8]
```diff
@@ -129,8 +135,8 @@ When Ignition runs on the first boot, and the decryption is successful, you will
 [    2.808874] coreos-secex-ignition-decrypt[740]: gpg: encrypted with rsa4096 key, ID <key_name>, created <yyyy-mm-dd>
 [  OK  ] Finished coreos-secex-igni…S Secex Ignition Config Decryptor.
 ----
-+
-If the decryption fails, you will see an output like the following example:
+
+* If the decryption fails, you will see an output like the following example:
```
Comment (agantony) [inline modules/ibm-z-secure-execution.adoc:139]:
```suggestion
** If the decryption failed, you can expect an output similar to the following example:
```

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

File: modules/ibm-z-secure-execution.adoc (hunk 6 above)
Comment (SNiemann15) [reply modules/ibm-z-secure-execution.adoc:125]: Will keep mention of fast-track installation. People tried verification before doing that.

File: modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc [hunk 9]
```diff
@@ -48,12 +48,14 @@ $ qemu-img create -f qcow2 -F qcow2 -b /var/lib/libvirt/images/{source_rhcos_qem
...
    --vcpus {vcpus} \
    --disk {disk} \
+   --launchSecurity type="s390-pv" \ <1>
    --import \
    --network network={network},mac={mac} \
-   --disk path={ign_file},format=raw,readonly=on,serial=ignition,startup_policy=optional <1>
+   --disk path={ign_file},format=raw,readonly=on,serial=ignition,startup_policy=optional <2>
 ----
-<1> If {ibm-name} Secure Execution is enabled, replace `serial=ignition` with `serial=ignition_crypted`.
+<1> If {ibm-name} Secure Execution is enabled, add the `launchSecurity type="s390-pv` parameter.
```
Comment (adellape) [inline modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc:60]:
```suggestion
<1> If {ibm-name} Secure Execution is enabled, add the `launchSecurity type="s390-pv"` parameter.
```

**Pull request merged**